docker-compose.yml

```diff
@@ -117,7 +117,7 @@ services:
                 testdata-gen --polka-btc-url 'ws://polkabtc:9944' --keyring=alice set-redeem-period --period=50
 
                 # the exchange rate value is 3855.23187 in FixedPoint format
-                testdata-gen --polka-btc-url 'ws://polkabtc:9944' --keyring=bob set-exchange-rate --exchange-rate=3855231870000000000000
+                testdata-gen --polka-btc-url 'ws://polkabtc:9944' --keyring=bob set-exchange-rate --exchange-rate=385523187
```

Comment:
Which version is correct, with the many 0s or without them?

Reply:
It used to work with many zeroes because of FixedPoint. This new parachain version broke many tests because vaults no longer had enough collateral. So I assume somethungs changed in how exchange rates are set